packages/kotti-ui/source/kotti-comment/components/CommentEntry.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+	<div class="kt-comment__entry">
+		<KtAvatar size="sm" :src="user.avatar" />
+		<div class="kt-comment__entry__wrapper">
+			<CommentHeader v-bind="headerProps" />
+			<CommentInlineEdit
+				v-bind="{
+					dangerouslyOverrideParser,
+					id,
+					isEditing,
+					message,
+					parentId,
+					postEscapeParser,
+				}"
+				@edit="onEdit"
+				@update:isEditing="isEditing = $event"
+			/>
+			<CommentActions
+				v-if="!isEditing"
+				v-bind="{ isDeletable, isEditable, showReply }"
+				@delete="onDelete"
+				@reply="onReply"
+				@update:isEditing="isEditing = $event"
+			/>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, ref } from '@vue/composition-api'
+import { pick } from 'lodash'
+
+import { makeProps } from '../../make-props'
+import { KottiComment } from '../types'
+
+import CommentActions from './CommentActions.vue'
+import CommentHeader from './CommentHeader.vue'
+import CommentInlineEdit from './CommentInlineEdit.vue'
+
+export default defineComponent<KottiComment.Entry.PropsInternal>({
+	name: 'CommentEntry',
+	components: {
+		CommentActions,
+		CommentHeader,
+		CommentInlineEdit,
+	},
+	props: makeProps(KottiComment.Entry.schema),
+	setup(props, { emit }) {
+		const isEditing = ref(false)
+
+		const isReply = computed(() => props.type === KottiComment.EntryType.REPLY)
+
+		return {
+			headerProps: computed(() => ({
+				...pick(props, ['createdAt', 'isModified', 'user']),
+				isInternalThread: !isReply.value ? props.isInternalThread : false,
```

Comment:
a reply inherits from its parent? why would you return false if it's not a reply

Reply:
Good catch, i don't need to do this.
So, CommentEntry renders a comment (either parent (first comment) or reply). According to design, only parent comments show `Internal thread` label in the header (the same for the `Reply` button).
Replies are already typed and they don't include isInternalThread, so it'll default to false and not show the `Internal thread` label in the header.
It's updated now.